nixos/modules/services/misc/nzbget.nix

```diff
@@ -93,6 +89,8 @@ in {
       '';
 
       serviceConfig = {
+        StateDirectory = dataDir;
```

Comment:
`StateDirectory` needs to be passed relative to `/var/lib/`. I'd just hardcode this to `nzbget`.